viewer/v2client/src/components/inspector/search-window.vue

```diff
@@ -330,33 +330,29 @@ export default {
       <template slot="panel-header-extra">
         <div class="SearchWindow-header search-header">
           <div class="SearchWindow-extractControls">
-            <p class="preview-entity-text uppercaseHeading">{{ "Your new entity" | translatePhrase }}:</p>
             <div class="copy-title" v-if="canCopyTitle">
               <label>
                 <input type="checkbox" name="copyTitle" v-model="copyTitle" /> 
                 {{ "Copy title from" | translatePhrase }} {{this.editorData.mainEntity['@type'] | labelByLang}}
               </label>
             </div>
           </div>
-          <div class="SearchWindow-summaryContainer">
-            <entity-summary 
-              :action-settings="localEntitySettings" 
-              :focus-data="itemInfo" 
-              :lines="4"
-              :should-link="false"></entity-summary>
-            <summary-action 
-              v-show="!extracting" 
-              :options="localEntitySettings" 
-              @action="extract()"></summary-action>
-          </div>
           <div class="SearchWindow-search search">
             <div class="SearchWindow-inputContainer input-container form-group panel">
               <input 
                 class="SearchWindow-input SearchWindowentity-search-keyword-input customInput form-control"
                 v-model="keyword"
                 autofocus
                 :placeholder="'Search' | translatePhrase">
-              <select 
+              <filter-select class="EntityAdder-filterSearchInput FilterSelect--insideInput"
+                :class-name="'js-filterSelect'"
+                :custom-placeholder="'All types:'"
+                :options="selectOptions"
+                :options-all="getRange"
+                :is-filter="true"
+                :options-selected="''"
+                v-on:filter-selected="setFilter($event, keyword)"></filter-select>
+              <!-- <select 
```

Comment:
Can we remove this commented out code? Is it used anymore? If not, just remove it. Or maybe we have a good reason to keep it?